utils/zerovec/src/ule/vec.rs

```diff
@@ -0,0 +1,42 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::ule::*;
+
+impl<T> AsVarULE for Vec<T>
```

Comment:
Could you add a convenience impl somewhere such that if you have `VZV<Vec<T::ULE>>`, you can get a `ZV<T>` out of it?  This should be possible by getting the slice and then using the `ZV::Borrowed` variant.

Reply:
(Will deal with in a followup)

Reply:
dealing with this now, actually